.ado/publish.yml

```diff
@@ -84,6 +84,10 @@ jobs:
         displayName: Beachball Publish (Stable Branch)
         condition: and(succeeded(), ne(variables['Build.SourceBranchName'], 'main'))
 
+      # Beachball reverts to local state after publish, but we want the updates it added
```

Comment:
What happens if a PR landed while the publish build was in progress?

Reply:
I think it is okay if we pull in something later than published. We are only consuming versions from package.json, along with changelog.json files. Both managed by beachball.